docs/API.AMQP.md

```diff
@@ -290,7 +333,9 @@ It works with the room unique ID Kuzzle returns to you when you make a subscript
 ```javascript
 {
   /*
-  Optional
+    Required. If your query doesn't include a requestId field, Kuzzle
```

Comment:
clientId instead of requestId, I guess ? ;)

anyway, is it really required, as we already have a "replyTo" header in the message that should be used to send back the query response ?